workflows/illumina.nf

```diff
@@ -94,6 +95,12 @@ workflow ILLUMINA {
     )
     ch_versions = ch_versions.mix(QUAST.out.versions)
 
+    // run prodigal on assemblies to predict ORFs and proteins
+    PRODIGAL (
+        ch_reformatted_assemblies, "gbk"
```

Comment:
what does gbk do? output in genbank format? what is the advantage to that?

Reply:
There are still fasta outputs, but you have to pick between gbk and gff or something